src/libraries/System.Drawing.Primitives/src/System.Drawing.Primitives.csproj

```diff
@@ -4,7 +4,7 @@
     <RootNamespace>System.Drawing</RootNamespace>
     <AssemblyName>System.Drawing.Primitives</AssemblyName>
     <DefineConstants Condition="'$(TargetsWindows)' == 'true'">$(DefineConstants);FEATURE_WINDOWS_SYSTEM_COLORS</DefineConstants>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)</TargetFrameworks>
```

Comment:
The test project's configuration is ridless, so you probably want to add the rid specific one there so that we don't loose Windows coverage? 
https://github.com/dotnet/runtime/blob/7c903f1556fe8ed9d32d8364013bb54df8d92edc/src/libraries/System.Drawing.Primitives/tests/System.Drawing.Primitives.Tests.csproj#L3